ui/packages/shared/components/src/GrpcMetadataContext/index.tsx

```diff
@@ -0,0 +1,29 @@
+import {grpc} from '@improbable-eng/grpc-web';
+import React from 'react';
+
+const EMPTY_METADATA = new grpc.Metadata();
+const GrpcMetadataContext = React.createContext<grpc.Metadata>(EMPTY_METADATA);
```

Comment:
maybe instead of using "React.createContext" and "React.useContext" below, we can just import the named "createContext" and "useContext"?

`import { createContext, useContext, ReactNode, ... } from 'react';

Reply:
Fixed it.